### PR TITLE
Remove reference to unused config value

### PR DESCRIPTION
**Pull request by justinbarclay**
Update documentation and example to reflect actual usage

### CODE REVIEW

File: unix/README.md [hunk 1]
```diff
@@ -132,10 +132,11 @@ You also can optionally capture metrics about processes running on the server:
 - Max Memory Used (MB)
 - Total Alive Time
 
-To enable capturing process metrics set the `gather_process_stats` key in `ansible.cfg` to `true`
+To enable capturing process metrics set the command-line flag for machine-stats `--process-stats` to `true`
 ```
-gather_process_stats = true
+machine-stats --process-stats true
```
Comment (przmv) [inline unix/README.md:137]: I added some debug output and it looks like `process_stats` is always true with:
- `--process-stats true`
- `--process-stats=true`
- `--process-stats false`
- `--process-stats=false`
- no `--process-stats` flag at all

Please check the [`argparse` documentation](https://docs.python.org/3/library/argparse.html) and search for `store_true` or `store_false`.

**Review by przmv — CHANGES_REQUESTED** [review]
Documentation is good, but the actual behavior is buggy.

**Comment by przmv** [comment]
Not related to this particular PR, but while running `machine-stats` with process stats enabled I got the error: `ERROR! petr@34.133.209.146: 'vmsize'`. Looking into the code, I discovered that there's no check for `vmsize` property to exist in the `status` dictionary.

**Comment by justinbarclay** [comment]
How do you actually run machine stats as you're developing it, in python? I am stuck. I used to just run __init__ but that no longer works. You give instructions on how to install dependencies locally but not how to run the app as your develping it.

**Comment by justinbarclay** [comment]
For reference I am trying: 
```
✦ ❯ python3 src/machine_stats/__init__.py --process-stats True --cpu-utilization-timeout 0
Traceback (most recent call last):
  File "/home/justin/dev/tidal/machine_stats/unix/src/machine_stats/__init__.py", line 19, in <module>
    os.environ[k] = v
  File "/usr/lib/python3.9/os.py", line 684, in __setitem__
    value = self.encodevalue(value)
  File "/usr/lib/python3.9/os.py", line 756, in encode
    raise TypeError("str expected, not %s" % type(value).__name__)
TypeError: str expected, not bool
```

**Comment by przmv** [comment]
> For reference I am trying:
> 
> ```
> ✦ ❯ python3 src/machine_stats/__init__.py --process-stats True --cpu-utilization-timeout 0
> Traceback (most recent call last):
>   File "/home/justin/dev/tidal/machine_stats/unix/src/machine_stats/__init__.py", line 19, in <module>
>     os.environ[k] = v
>   File "/usr/lib/python3.9/os.py", line 684, in __setitem__
>     value = self.encodevalue(value)
>   File "/usr/lib/python3.9/os.py", line 756, in encode
>     raise TypeError("str expected, not %s" % type(value).__name__)
> TypeError: str expected, not bool
> ```

This was fixed in https://github.com/tidalmigrations/machine_stats/commit/44dd3b9ea018d34eeac84eb3018a7ca451cee736

**Comment by przmv** [comment]
> How do you actually run machine stats as you're developing it, in python? I am stuck. I used to just run **init** but that no longer works. You give instructions on how to install dependencies locally but not how to run the app as your develping it.

1—3 are prerequisites, 4—5 are for actual development and running the dev version

1. Install `pipenv`
2. `cd unix`
3. `pipenv install && pipenv install --dev`
4. `vi src/machine_stats/__init__.py`
5. `pipenv run machine_stats --help`

**Comment by jmariomejiap** [comment]
> How do you actually run machine stats as you're developing it, in python? I am stuck. I used to just run **init** but that no longer works. You give instructions on how to install dependencies locally but not how to run the app as your develping it.

We should adjust the instructions to include how to run it locally. Can you please @przmv update documentation?

**Comment by justinbarclay** [comment]
Awesome, I am able to run the init file directly now 😄

**Comment by przmv** [comment]
> > How do you actually run machine stats as you're developing it, in python? I am stuck. I used to just run **init** but that no longer works. You give instructions on how to install dependencies locally but not how to run the app as your develping it.
> 
> We should adjust the instructions to include how to run it locally. Can you please @przmv update documentation?

Submitted a new PR https://github.com/tidalmigrations/machine_stats/pull/110

**Comment by justinbarclay** [comment]
> Not related to this particular PR, but while running `machine-stats` with process stats enabled I got the error: `ERROR! petr@34.133.209.146: 'vmsize'`. Looking into the code, I discovered that there's no check for `vmsize` property to exist in the `status` dictionary.

Thats weird, I found nothing to indicate that the vmsize property might not exist in the status file. The closest to a reference that I could find is [here](https://www.kernel.org/doc/html/latest/filesystems/proc.html). @przmv I'll fix this in a subsequent MR, but do you know if it's safe to assume any of these fields would exist in the status file?

**Comment by justinbarclay** [comment]
and here is a video demonstrating the changes to be... uh closer to ~our expectations~ working

https://user-images.githubusercontent.com/6868388/142241579-f3e2fe50-1987-48ca-a183-c1ec90eb7536.mp4

**Review by przmv — APPROVED** [review]
LGTM!

**Pull request merged**